examples/tcl_rockstar.py

```diff
@@ -0,0 +1,6 @@
+from rockstar import RockStar
+
+tcl_code = """puts "Hello, World!"
+"""
```

Comment:
any reason for new line? I think following will be fine:

    'puts "Hello, World!"'